From cd3723ab3d99b7ae8be69fe05a04afe615074932 Mon Sep 17 00:00:00 2001
From: Mark Salyzyn <salyzyn@google.com>
Date: Tue, 9 Feb 2016 12:31:42 -0800
Subject: [PATCH 691/769] android: recommended.cfg: enable taskstats

CONFIG_TASKSTATS=y
CONFIG_TASK_DELAY_ACCT=y
CONFIG_TASK_XACCT=y
CONFIG_TASK_IO_ACCOUNTING=y

Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Bug: 21334988
Bug: 26966375
Change-Id: I17b097ae4ea6c63c2e9fddd9544e3f06d49b609d
---
 android/configs/android-recommended.cfg | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/android/configs/android-recommended.cfg b/android/configs/android-recommended.cfg
index 960b9de..4dd3b45 100644
--- a/android/configs/android-recommended.cfg
+++ b/android/configs/android-recommended.cfg
@@ -109,6 +109,10 @@ CONFIG_TABLET_USB_GTCO=y
 CONFIG_TABLET_USB_HANWANG=y
 CONFIG_TABLET_USB_KBTAB=y
 CONFIG_TABLET_USB_WACOM=y
+CONFIG_TASKSTATS=y
+CONFIG_TASK_DELAY_ACCT=y
+CONFIG_TASK_IO_ACCOUNTING=y
+CONFIG_TASK_XACCT=y
 CONFIG_TIMER_STATS=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
-- 
1.9.1

